training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -76,7 +76,7 @@
     "\n",
```

Comment:
<p>In the itemized list, we stated the following:</p><p> </p><blockquote><code>train_gpt_simple.py</code>: This is an entrypoint script that is passed to the Pytorch estimator in the notebook instructions. This script is responsible for end to end training of the GPT-2 model with SMP. <strong>The script has additional comments at places where the SMP API is used.</strong></blockquote><p> </p><p>Therefore, in the training script, please make sure adding commentary lines to everywhere you have SMP modifications.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Comments were already added but were marked with smdistributed tag, I've updated the comments to all have the same tag of 'SMP modification'.